phpBB/includes/compatibility_globals.php

```diff
@@ -48,6 +48,9 @@ function register_compatibility_globals()
 	/* @var $user \phpbb\user */
 	$user = $phpbb_container->get('user');
 
+	/* @var \phpbb\language\language $language */
+	$language = $phpbb_container->get('language');
```

Comment:
Nothing relevant, though I'd put this one before `$phpbb_log = $phpbb_container->get('log');`

Reply:
I put it next to `$user` for historic reasons.

Reply:
https://github.com/phpbb/phpbb/pull/5513/files/0e38bebaa56dcf8921df99e8eb85e3046721dd13#r276113152